config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml

```diff
@@ -688,3 +709,32 @@ periodics:
       env:
       - name: NETWORK_POLICY_PROVIDER
         value: calico
+
+- interval: 2h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-containerd-gce-netd
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-kubernetes-e2e-enable-netd: "true"
+    preset-kubernetes-e2e-containerd: "true"
```

Comment:
I added one more preset labels, will this work? @krzyzacy

Reply:
as long as it's not conflicting with other preset labels :-)